src/fibonacciBases.js

```diff
@@ -0,0 +1,43 @@
+function fibonacci(n) {
+  return n === 0 || n === 1 ? 1 : fibonacci(n - 2) + fibonacci(n - 1)
+}
+
+function decToFibonnaci(number) {
```

Comment:
I like the naming conventions you chose. I noticed everyone had a similar thought process when it came to naming variables for fibonacci :)